packages/geojson-extension/src/access_data.json

```
@@ -0,0 +1,29 @@
+{
+  "Thunderforest": {
+    "keyString": "apikey"
+  },
```

Then maybe no need for `"keyString"`?
```suggestion
  "Thunderforest": "apikey",
```

---

keyString has been removed.